rpnpy/functions.py

```diff
@@ -169,6 +169,7 @@ def join(calc, modifiers, count):
     @param count: An C{int} count of the number of arguments to pass.
     """
     sep, args = calc.findStringAndArgs('join', modifiers, count)
+    print(f'sep is {sep}, args is {args!r}')
```

Comment:
Seems there's another debug print here, just in case you missed it.

Reply:
Yep, got that one too - thanks!